tests/api-testing/tests/test_foldersnew.py

```
@@ -128,12 +199,62 @@
+        f"{resp_create_new_alerts_folders_blank.content}"
+    )
+
+def test_invalid_folder_id_retrieve_alerts(create_session, base_url):
+    org_id = "default"
+    invalid_folder_id = "e56454545dfd@43232"  # Folder ID that doesn't exist
+    
+    resp_get_alerts_folders_invalid = create_session.get(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{invalid_folder_id}"
+    )
+    assert resp_get_alerts_folders_invalid.status_code == 404, (
+        f"Expected 404 Not Found, but got {resp_get_alerts_folders.status_code} "
```

Ruff (0.8.2)

Undefined name `resp_get_alerts_folders`

(F821)